package.json

```diff
@@ -10,6 +10,7 @@
     "node": ">=8.6.0"
   },
   "description": "A json wrapper around publicly open data in Iceland",
+  "licence": "MIT",
```

Comment:
Should be "license", right?

Reply:
[I think both are applicable]I(https://docs.npmjs.com/files/package.json#files)

Reply:
I see no other mention of "licence" on that page other than referring to the name of the file called LICENCE (you named the file LICENSE anyway, so that doesn't apply).

I think the ["license" key in package.json](https://docs.npmjs.com/files/package.json#license) has to be spelled correctly though? There is at least no mention of the "licence" spelling?